github3/gists/file.py

```diff
@@ -31,7 +31,16 @@ def _update_attributes(self, attributes):
         #: The size of the file.
         self.size = attributes.get('size')
         #: The content of the file.
-        self.content = attributes.get('content')
+        self.original_content = attributes.get('content')
 
     def _repr(self):
         return '<Gist File [{0}]>'.format(self.name)
+
+    def content(self):
+        """Retrieve contents of file from key 'raw_url' if there is no
+        'content' key in Gist object.
+        """
+        resp = self._get(self.raw_url)
+        if self._boolean(resp, 200, 404):
```

Comment:
i like that suggestion. @sigmavirus24 is the one who recommended the implementation...if he is ok with it, then i will change to ur suggestion. thank you for the review :)